### PR TITLE
Build fixes after android awt split

### PR DESCRIPTION
**Pull request by AlexeyTsvetkov**

### CODE REVIEW

**Comment by olonho** [comment]
Nice!

File: skiko/build.gradle.kts [hunk 1]
```diff
@@ -1068,12 +1104,13 @@ publishing {
         allJvmRuntimeJars.forEach { entry ->
             val os = entry.key.first
             val arch = entry.key.second
-            create<MavenPublication>("skikoJvmRuntime${targetId(os, arch)}") {
+            create<MavenPublication>("skikoJvmRuntime${toTitleCase(os.id)}${toTitleCase(arch.id)}") {
```
Comment (olonho) [inline skiko/build.gradle.kts:1107]: Not sure if I like this particular change. Readability seems better when target is clearly marked, and this captializing just makes readability worse.

File: skiko/buildSrc/src/main/kotlin/utils.kt [hunk 2]
```diff
@@ -16,6 +18,21 @@ inline fun <reified T : Task> TaskContainer.registerOrGetTask(
...
+    crossinline fn: T.() -> Unit
+): TaskProvider<T> = tasks.register(taskName, T::class) { fn() }
+
+inline fun <reified T : Task> Project.registerSkikoTask(
+    actionName: String,
+    targetOs: OS,
+    targetArch: Arch,
+    crossinline fn: T.() -> Unit
+): TaskProvider<T> {
+    val taskName = joinToTitleCamelCase(actionName, targetOs.id, targetArch.id)
```
Comment (olonho) [inline skiko/buildSrc/src/main/kotlin/utils.kt:32]: Same here.

File: skiko/build.gradle.kts (hunk 1 above)
Comment (AlexeyTsvetkov) [reply skiko/build.gradle.kts:1111]: What's the point of readability, when different cases are mixed in inconsistent and unpredictable way?
If we could change the default case format for all tasks, I would be pro dash case.
But that's not possible, and lack of some predictable naming convention and style seriously bothers me

Comment (olonho) [reply skiko/build.gradle.kts:1111]: OK

**Review by olonho — APPROVED** [review]

**Pull request merged**